staging/src/k8s.io/kubectl/pkg/cmd/scale/scale.go

```
@@ -209,7 +209,7 @@ func (o *ScaleOptions) RunScale() error {
 	}
 
 	infos := []*resource.Info{}
-	err = r.Visit(func(info *resource.Info, err error) error {
+	_ = r.Visit(func(info *resource.Info, err error) error {
```

should this error be ignored?